spot_driver/spot_driver/spot_ros2.py

```diff
@@ -2574,6 +2577,27 @@ def arm_joint_cmd_callback(self, data: JointState) -> None:
 
         self.spot_wrapper.arm_joint_cmd(**arm_joint_map)
 
+    def arm_pose_cmd_callback(self, data: PoseStamped) -> None:
+        if not self.spot_wrapper:
+            self.get_logger().info(f"mock mode, received arm pose command {data}")
```

Comment:
the most ultra nitpick (for consistency)

```suggestion
            self.get_logger().info(f"Mock mode, received arm pose command {data}")